### ISSUE TITLE
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple[BUG]

### ISSUE THREAD

**Issue by rishabh3354** [pytube/pytube#1104]
![Screenshot from 2021-08-27 08-50-05](https://user-images.githubusercontent.com/26843758/131066773-4a22daec-5c47-44e1-90d7-e9297cc102b4.png)

Not able to download videos and playlist

i am using pytube 11.0
system: Ubuntu 20.04

**Comment by truegav** [comment]
Similar error when trying to get streams.
```
>>> from pytube import YouTube
>>> yt = YouTube("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
>>> yt.streams
```
gives
```
Traceback (most recent call last):
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/__main__.py", line 170, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/__main__.py", line 285, in streams
    return StreamQuery(self.fmt_streams)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/__main__.py", line 177, in fmt_streams
    extract.apply_signature(stream_manifest, self.vid_info, self.js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/extract.py", line 409, in apply_signature
    cipher = Cipher(js=js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 43, in __init__
    self.throttling_plan = get_throttling_plan(js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 387, in get_throttling_plan
    raw_code = get_throttling_function_code(js)
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 293, in get_throttling_function_code
    name = re.escape(get_throttling_function_name(js))
  File "/home/hacker097/.local/lib/python3.8/site-packages/pytube/cipher.py", line 278, in get_throttling_function_name
    raise RegexMatchError(
pytube.exceptions.RegexMatchError: get_throttling_function_name: could not find match for multiple
```

**Comment by tranthanhbinh2603** [comment]
I have the solution. First, go to the address:
/home/hacker097/.local/lib/python3.8/site-packages/pytube/

Then find the file cipher.py.
Edit this line: r'a.[AZ]&&(b=a.[sg]et("n"))&&(b=([^(]+)(b) ,a.set("n",b))}};',

To: r'a.[AZ]&&(b=a.[sg]et("n"))&&(b=([^(]+)(b),a .[sg]et("n",b))}};',

Reference from: xioren@35a6536

**Comment by rishabh3354** [comment]
Fixed with pip package (temperory basic you can use it with pip)

pip install git+https://github.com/rishabh3354/pytube@master

**Comment by tfdahlin** [comment]
Resolved in v11.0.1, please update with `python -m pip install --upgrade pytube`. Locking this thread as it is a duplicate of #1100.
